ui/src/reducers/index.js

```diff
@@ -74,7 +104,7 @@ export default (state = initialState, action) => {
     case APP_VERSIONS_SORT:
       const versionDirection = action.payload.direction;
       const versionIndex = action.payload.index;
-      const sortedAppVersions = sortRows(state.app.versionsRows, versionIndex, versionDirection);
+      const sortedAppVersions = sortRows(state.app.versionsRows, versionIndex, versionDirection, state.appVersionsSortDirection.index, state.appVersionsSortDirection.direction);
```

Comment:
@craicoverflow why are these additional values added here?

Reply:
woops I put them in when I was testing a way to fix this but forgot to take em out! will remove.